.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Use local JWTKit
         run: swift package --package-path jwt edit jwt-kit --path ./jwt-kit
       - name: Run tests with Thread Sanitizer
-        run: swift test --package-path jwt --sanitize=thread
+        run: swift test --package-path jwt --sanitize=thread -Xswiftc -warnings-as-errors
```

Comment:
This is a good way to make sure CI never ever passes, given the way warnings tend to show up without warning (no pun intended) thanks to changes in dependencies.